extensions/wikia/UserProfilePageV3/UserIdentityBox.class.php

```diff
@@ -20,6 +20,8 @@ class UserIdentityBox {
 	const USER_OCCUPATION_CHAR_LIMIT = 200;
 	const USER_GENDER_CHAR_LIMIT = 200;
 
+	const CACHE_TTL = 60 * 60 * 24; // 24 hours
```

Comment:
User data on profile pages used to be cached indefinitely until the next update. With the discussion apps, "location" can now be updated which is part of that cached data. I added a TTL so that we would have eventual consistency in the case of an update by the discussion app.

I'm being kind of conservative here and picking 24 hours to reduce db hits and requests to the attribute service. That may be too long though.